libs/model/src/user/GetNewContent.query.ts

```diff
@@ -0,0 +1,74 @@
+import { Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { Op, QueryTypes } from 'sequelize';
+import { models, sequelize } from '../database';
+
+export function GetNewContent(): Query<typeof schemas.GetNewContent> {
+  return {
+    ...schemas.GetNewContent,
+    auth: [],
+    body: async ({ actor }) => {
+      const joinedCommunityIdsWithNewContent: string[] = [];
+
+      // get addresses for auth user
+      const addresses = await models.Address.findAll({
+        where: {
+          user_id: actor.user.id!,
+          verified: { [Op.not]: null },
+          last_active: { [Op.not]: null },
+        },
+      });
+
+      // find if there is new content for each community per user address
+      for (let i = 0; i < addresses.length; i++) {
```

Comment:
The new query looks good, but I think you can optimize the internal SQL by joining to community/addresses instead of looping

Reply:
mb, you also mentioned this before but I missed this, will update tomorrow first thing

Reply:
done https://github.com/hicommonwealth/commonwealth/pull/8628/commits/ee174ab93113be46edcb5f099f991518536e0688, plz lmk if any issues